packages/x-data-grid/src/components/cell/GridActionsCell.tsx

```diff
@@ -322,6 +331,9 @@ GridActionsCell.propTypes = {
   value: PropTypes.any,
 } as any;
 
+GridActionsCell.Item = GridActionsCellItem as React.FC<GridActionsCellItemProps>;
```

Comment:
Had to add `as React.FC<GridActionsCellItemProps>;` here to fix this error:

```
src/components/cell/GridActionsCell.tsx:334:1 - error TS4032: Property 'Item' of exported interface has or is using name 'GridActionsCellItemCommonProps' from private module '"/tmp/e7c18820/packages/x-data-grid/src/components/cell/GridActionsCellItem"'.

334 GridActionsCell.Item = GridActionsCellItem;
```

https://ci.codesandbox.io/status/mui/mui-x/pr/15041/builds/553072